src/features/script-load.js

```diff
@@ -46,3 +46,17 @@ systemJSPrototype.instantiate = function (url, firstParentUrl) {
   });
 };
 
+if (typeof window !== 'undefined') {
```

Comment:
If we attach events to document we can use the shared `typeof document` check (I think it's refactored to `hasDocument` in common), to save a byte or two.